packages/common/src/services/solana-client/SolanaClient.ts

```diff
@@ -22,7 +22,9 @@ export class SolanaClient {
     try {
       this.connection = new Connection(solanaClusterEndpoint!, 'confirmed')
     } catch (e) {
-      console.error('Could create Solana RPC connection', e)
+      if (process.env.JEST_WORKER_ID === undefined) {
```

Comment:
fun! just wondering what the idea is here? is this to avoid extra logs when running tests? would this be the way we would handle logs like this going forward? Would we want to just mock out SolanaClient?

Reply:
Yeah this was to avoid big errors when testing, would probably be better to mock solana client. Let me try that!

Reply:
Also curious about using JEST_WORKER_ID? I thought JEST sets `NODE_ENV='test'`. (https://jestjs.io/docs/environment-variables)